tests/dash/test_dash_privatelink.py

```diff
@@ -78,6 +79,8 @@ def common_setup_teardown(localhost, duthost, ptfhost, dpu_index, skip_config):
     if skip_config:
         return
 
+    global hwsku
```

Comment:
would prefer to avoid global vars when possible. suggest calling the `config_facts` fixture directly from the test function and reading HWSKU from there or pulling out HWSKU detection/checking into a new fixture entirely.